app/server-routes/src/main/scala/org/bitcoins/server/routes/Server.scala

```diff
@@ -119,8 +130,10 @@ case class Server(
   private val eventsRoute = "events"
 
   private def wsRoutes: Route = {
-    path(eventsRoute) {
-      Directives.handleWebSocketMessages(wsHandler)
+    authenticateBasic("auth", authenticator) { _ =>
```

Comment:
Whats the benefit of adding auth on the websocket? Currently we only write things to the websocket and do not support requests sent to us over the websocket.

Reply:
Same benefit as for the HTTP server - it adds access control.